net/epee-encoding/src/macros.rs

```diff
@@ -35,7 +35,7 @@ macro_rules! try_right_then_left {
 macro_rules! epee_object {
     (
         $obj:ident,
-        $($field: ident $(($alt_name: literal))?: $ty:ty $(as $ty_as:ty )? $(= $default:expr)?  $(=> $read_fn:expr, $write_fn:expr, $should_write_fn:expr)?, )+
+        $($field: ident $(($alt_name: literal))?: $ty:ty $(as $ty_as:ty )? $(= $default:expr)?  $(=> $read_fn:expr, $write_fn:expr, $should_write_fn:expr)?, )*
```

Comment:
Split into multiple lines? Maybe some `// comments` explaining what each input is too.

Reply:
I didn't split this into multiple lines but I did add docs to this macro.